PkFokam-Alumni-Network/CommunityWebsite
# Change website name to PKFOkam Alumni Society

## WHY

This change is required to update the website name to "PKFOkam Alumni Society" as per the new branding guidelines. This ensures that the website reflects the correct name and maintains consistency across all pages.

## WHAT

I have updated the website name in the following files:
- `index.html`: Changed the title tag to "PKFOkam Alumni Society".
- `Header.jsx`: Updated the header text to "PKFOkam Alumni Society".
- `Banner.jsx`: Modified the banner text to "Welcome to PKFOkam Alumni Society".

## TESTING

To test these changes:
1. Open the website in a browser.
2. Verify that the title in the browser tab is "PKFOkam Alumni Society".
3. Check the header section to ensure it displays "PKFOkam Alumni Society".
4. Look at the banner section to confirm it says "Welcome to PKFOkam Alumni Society".

No unit tests are required for these changes as they are purely textual updates. Visual inspection is sufficient to verify the correctness.
